### ISSUE TITLE
Community context menu should not say "Leave community" if not joined

### ISSUE THREAD

**Issue by 0x-r4bbit** [status-im/status-desktop#10963]
Here's a screenshot of a spectated community I'm looking at.
Notice the context menu I've opened using right-click on the community item in the app navbar.
The context menu has an option to "Leave community", which will remove the community from the list.
While the functionality is what I'm aiming for, semantically, I'm not actually leaving the community (I haven't even joined yet).
Might make sense to rename this for spectated communities. Maybe something like "Remove"? /cc @John-44 

![Screenshot from 2023-06-07 09-18-00](https://github.com/status-im/status-desktop/assets/445106/f3f68626-e14b-4982-9e94-a9ff884c6772)

**Comment by John-44** [comment]
@0x-r4bbit because the user hasn't joined the community at this point, the red option in this context menu should use the wording "Close Community".

Of course if the user *is* a member of the community, the wording should remain as "Leave Community"

**Comment by caybro** [comment]
Can we also add a confirmation dialog when really leaving? I accidentally left my local test community the other day without noticing

**Comment by benjthayer** [comment]
Hi @0x-r4bbit @caybro , @John-44 and I went over a few new "Leave Community" flows to address this issue and that raised by @caybro 

The designs covers:

- Closing Communities that the user is spectating (not yet joined)
- Leaving Communities the user is a member of (this features a confirmation dialog which features the Community Owner's leave message which is defined via the Community creation flow)

With regards to owners leaving Communities (both in the case that they are or aren't currently on a device that is the control node for the Community), we are proposing to remove the `Leave Community` action from the right-click menu for now given it has such serious repercussions. The introduction of the new Community Owner Token which is being worked on this week will allow the user to transfer ownership of the Community by sending the token to another user. So that new flow needs to be completed before we can consider what aspects of it should be presented when an Owner goes to leave their own Community. 


Designs
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?type=design&node-id=31451-561251&t=bO2xh3Cg3mbrnUey-4

**Comment by caybro** [comment]
I'll take this, for 0.14 right?

**Comment by John-44** [comment]
@caybro see all the updated designs that @benjthayer has done in the Desktop Settings figma here https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=16089%3A387522&t=7akjEGYTrHADlQRv-1

Below is just one screenshot from these new design, but really I recommend looking at the figma linked above for the full context and all the flows:

![2023-06-13_13-47-30](https://github.com/status-im/status-desktop/assets/22846356/814c19c7-dedb-402a-839c-77e75b051c16)

**Comment by caybro** [comment]
> @caybro see all the updated designs that @benjthayer has done in the Desktop Settings figma here https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?type=design&node-id=16089%3A387522&t=7akjEGYTrHADlQRv-1
> 
> Below is just one screenshot from these updated designs, but really I recommend looking at the figma linked above for the full context and all the flows:
> 
> ![2023-06-13_13-47-30](https://user-images.githubusercontent.com/22846356/245480599-814c19c7-dedb-402a-839c-77e75b051c16.png)

OK, this will be another issue/task then; I just figured we have the same "leave" action here; luckily this one already has the confirmation dialog, so I'll just reuse it :)

![image](https://github.com/status-im/status-desktop/assets/5377645/fb245f9f-d356-4d28-b189-ce8e01864b72)

**Comment by caybro** [comment]
Just saving it here.... another instance of the "leave" item:

![image](https://github.com/status-im/status-desktop/assets/5377645/768cffd7-1d52-404e-bbbe-12bbb5dcd649)

**Comment by jrainville** [comment]
> I'll take this, for 0.14 right?

Yes. Not that big of an issue + not much time left in 0.13

**Comment by John-44** [comment]
> Just saving it here.... another instance of the "leave" item:
> 
> ![image](https://user-images.githubusercontent.com/5377645/245502175-768cffd7-1d52-404e-bbbe-12bbb5dcd649.png)

I think this dialogue gets killed with the new community private key transfer designs.  So to fix in 0.14, which starts next week ;-)

**Comment by caybro** [comment]
Reopening since it got broken after 82a1ed2f7aa46750fb5ab891e2ce40f61287eccb had been merged (it removed the `amIMemberAdmin` role) :/

**Comment by caybro** [comment]
@iurimatias @alexandraB99 worth targetting 0.13RC?

**Comment by jrainville** [comment]
> @iurimatias @alexandraB99 worth targetting 0.13RC?

RC issues should be for actual blockers for the RC, so this shoudl stay in 0.14. 

It's fine that you asked. You can check the release process here: https://www.notion.so/Desktop-app-Release-process-356bbe7c40d14db3b64ab364800a48c9
